client/src/actions/index.js

```diff
@@ -135,7 +135,15 @@ const doInitialDataLoad = () =>
       Step 3 - load everything else
       */
       await obsAnnotationFetchAndLoad(dispatch, schema, universe);
-
+      const firstCategory = (schema.schema.annotations?.obs?.columns || []).find(
```

Comment:
I don't think this is an appropriate test to find "first", as it does not use the same logic as the category component.  It will miss non-categorical annotations which are displayed as categorical (ie, string, bool), and there is currently no guarantee that the schema order will match the render order.

Please see src/components/categorical/categorical.js for the actual logic:
```
    /* all names, sorted in display order.  Will be rendered in this order */
    const allCategoryNames = ControlsHelpers.selectableCategoryNames(
      schema,
      ControlsHelpers.maxCategoryItems(config)
    ).sort();
```

Reply:
True. I realized this after putting up the PR and then tagged it with `[WIP]`. Will figure this out in a bit.